pkg/k8s/watchers/watcher.go

```diff
@@ -786,6 +786,8 @@ func (k *K8sWatcher) GetStore(name string) cache.Store {
 		return k.networkpolicyStore
 	case "namespace":
 		return k.namespaceStore
+	case "pod":
+		return k.podStore
```

Comment:
```
	<-k.podStoreSet
	k.podStoreMU.RLock()
	defer k.podStoreMU.RUnlock()
        return k.podStore
```

Reply:
@aanm i updated the PR. kind of curious why we need `k.podStoreMU`. looks like k.podStore gets set once during `podInit()` and never gets modified. isn't it sufficient to block on `k.podStoreSet`?

Reply:
@michi-covalent with a kvstore setup + `enable-k8s-event-handover` the `podStore` variable can be set multiple times depending on the connectivity with the kvstore.